agave/models/helpers.py

```diff
@@ -1,9 +1,49 @@
+import datetime as dt
 import uuid
 from base64 import urlsafe_b64encode
+from typing import Any
+
+from rom import Column, Model, PrimaryKey
 
 
 def uuid_field(prefix: str = ''):
     def base64_uuid_func() -> str:
         return prefix + urlsafe_b64encode(uuid.uuid4().bytes).decode()[:-2]
 
     return base64_uuid_func
+
+
+class String(Column):
+    """
+    No utilizo la clase String de rom porque todo lo maneja en bytes
+    codificado en latin-1.
+    """
+
+    _allowed = str
+
+    def _to_redis(self, value):
+        return value.encode('utf-8')
+
+    def _from_redis(self, value):
+        return value.decode('utf-8')
+
+
+def sanitize_item(item: Any) -> Any:
+    if isinstance(item, dt.date):
+        rv = item.isoformat()
+    elif hasattr(item, 'dict'):
+        rv = item.dict()
+    else:
+        rv = item
+    return rv
```

Comment:
@felipaoo I thought we agreed to use the one from `cuenca-validations`

Reply:
Sip se va a agregar en cuenca-validations, solo lo puse para probar redis, este código no estará aqui.

Reply:
Yes, we are going to take the one from `cuenca-validations` and remove it

Reply:
Listo el pr en cuenca-validations https://github.com/cuenca-mx/cuenca-validations/pull/74